dev/doc/release-process.md

```diff
@@ -20,10 +20,12 @@
 ## On the branching date ##
 
 - [ ] Merge the above PR and create the `vX.X` branch from the last merge commit before this one (using this name will ensure that the branch will be automatically protected).
-- [ ] Set the next major version alpha tag using `git tag -s`.  The `VY.Y+alpha` tag marks the first commit to be in `master` and not in the `vX.X` release branch. Note that this commit is the first commit in the first PR merged in master, not the merge commit for that PR. Therefore, if you proceeded as described above, this should be the commit updating the version, magic numbers and compatibility infrastructure.  After tagging, double-check that `git describe` picks up the tag you just made (if not, you tagged the wrong commit).
+- [ ] Set the next major version alpha tag using `git tag -s` (you can leave the tag message empty).  The `VY.Y+alpha` tag marks the first commit to be in `master` and not in the `vX.X` release branch (be careful about small `v` for branches and big `V` for tags). Note that this commit is the first commit in the first PR merged in master, not the merge commit for that PR. Therefore, if you proceeded as described above, this should be the commit updating the version, magic numbers and compatibility infrastructure.  After tagging, double-check that `git describe` picks up the tag you just made (if not, you tagged the wrong commit).
 - [ ] Push the new tag with `git push upstream VY.Y+alpha --dry-run` (remove the `--dry-run` and redo if everything looks OK).
-- [ ] Start a new ["classic" project](https://github.com/coq/coq/projects?type=classic) to track PR backporting. The project should have a `Request X.X+rc1 inclusion` column for the PRs that were merged in `master` that are to be considered for backporting, and a `Shipped in X.X+rc1` columns to put what was backported. A message to `@coqbot` in the milestone description tells it to automatically add merged PRs to the `Request ... inclusion` column and backported PRs to the `Shipped in ...` column. See previous milestones for examples. When moving to the next milestone (e.g. `X.X.0`), you can clear and remove the `Request X.X+rc1 inclusion` column and create new `Request X.X.0 inclusion` and `Shipped in X.X.0` columns.
+- [ ] In the milestone, add to the description a line like `@coqbot: backport to v8.20 (move rejected PRs to: https://github.com/coq/coq/milestone/60)` (in case PRs must be backported to multiple (point releases) branches, something like `@coqbot: backport to v8.19 (move rejected PRs to: <url of current milestone for 8.20>); backport to v8.20 (move rejected PRs to: <URL of current milestone for 8.21>); ...`)
```

Comment:
Maybe the part about the milestone and the backporting project should be moved above the part about the tag because, while the tag can wait without creating trouble, if the milestone description is not set as soon as the new branch is created, there is a risk of missing out some PRs that would be merged in the meantime.

Reply:
I don't understand the part about having several backport directives. If we also write `backport to v8.19` inside the description of 8.20.1 milestone, this is bound to drive a lot of people crazy.

Reply:
I think the use case is rather the contrary: something tagged with milestone 8.19.3 should: backport to 8.19.3; backport to 8.20.1
@Zimmi48 is it the use case you have in mind, if yes, shouldn't we rather recommend when creating a milestone to add the "backport to <new_milestone>" to all still open milestone for minor versions of previous releases?

Reply:
Oh, that makes a lot more sense when you word it that way!

Reply:
Indeed, this has happened before that this was needed. And yes, @proux01 I think adding the recommendation you mentioned would be worthwhile. (There have also been cases before where milestones like this were created, but this step was forgotten, and because of this some backports were missed).

Reply:
@Zimmi48 done, if you're happy with it, feel free to assign and merge